Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "languages-github-actions"
+description = "GitHub Actions for the Languages Team"
+version = "0.0.1"
+repository = "https://github.com/heroku/languages-github-actions.git"
+rust-version = "1.66"
+edition = "2021"
+publish = false
+
+[[bin]]
+name = "actions"
+path = "src/main.rs"
+
+[dependencies]
+chrono = "0.4.24"
+clap = { version = "4.2.1", default-features = false, features = [
+    "derive",
+    "error-context",
+    "help",
+    "std",
+    "usage",
+] }
+indexmap = "1.9.3"
+lazy_static = "1.4.0"
+libcnb-data = { git = "https://github.com/heroku/libcnb.rs", branch = "feature/W-12731729_libcnb_package" }
+libcnb-package = { git = "https://github.com/heroku/libcnb.rs", branch = "feature/W-12731729_libcnb_package" }
```

Comment:
I presume this is on your list already, but once there is a new libcnb.rs release, the Git dependency here should be changed to crates.io

Reply:
Resolved in d470f06179e81039d730289e69a3290f9620bdd9